packages/expo-permissions/ios/EXPermissions/EXPermissions.h

```diff
@@ -3,31 +3,22 @@
 #import <UMCore/UMExportedModule.h>
 #import <UMCore/UMModuleRegistryConsumer.h>
 #import <UMPermissionsInterface/UMPermissionsInterface.h>
-#import <EXPermissions/EXPermissionBaseRequester.h>
 
 FOUNDATION_EXPORT NSString * const EXPermissionExpiresNever;
 
-typedef enum EXPermissionStatus {
-  EXPermissionStatusDenied,
-  EXPermissionStatusGranted,
-  EXPermissionStatusUndetermined,
-} EXPermissionStatus;
+@interface EXPermissions : UMExportedModule <UMPermissionsInterface, UMModuleRegistryConsumer>
 
-@protocol EXPermissionsModule
-
-- (dispatch_queue_t)methodQueue;
-
-@end
-
-@interface EXPermissions : UMExportedModule <EXPermissionRequesterDelegate, UMPermissionsInterface, UMModuleRegistryConsumer, EXPermissionsModule>
+// TODO: Remove once SDK34 is phased out
+- (void)askForGlobalPermission:(NSString *)permissionType
+                  withResolver:(void (^)(NSDictionary *))resolver
```

Comment:
I wonder whether it can just be 
```suggestion
                  withResolver:(UMPromiseResolveBlock)resolver
```